### PR TITLE
Smartsheet cert

### PR DESCRIPTION
**Pull request by work-insights-data**

### CODE REVIEW

**Comment by Court72** [comment]
@DougKlopfenstein

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by work-insights-data** [comment]
@DougKlopfenstein @Court72 
The new connector has been shipped, users need this documentation to be merged so they have help available to them.
When will this be merged?

File: powerquery-docs/connectors/smartsheet.md [hunk 1]
```diff
@@ -15,18 +15,54 @@ ms.author: dougklo
 | ---- | ----------- |
 | Release State | General Availability |
```
Comment (ptyx507x) [inline powerquery-docs/connectors/smartsheet.md:16]: This connector is not in general availability. Please change this to be in Beta and match the current status of your connector

File: powerquery-docs/connectors/smartsheet.md [hunk 2]
```diff
@@ -15,18 +15,54 @@ ms.author: dougklo
...
 | Products | Power BI (Semantic models)<br/>Power BI (Dataflows)<br/>Fabric (Dataflow Gen2) |
```
Comment (ptyx507x) [inline powerquery-docs/connectors/smartsheet.md:17]: it should read "Fabric Dataflow Gen2" and "Power BI Dataflow"

File: powerquery-docs/connectors/smartsheet.md [hunk 3]
```diff
@@ -15,18 +15,54 @@ ms.author: dougklo
...
+
+8. Once successfully logged in, select **Connect**.
+
+   ![Signed in and ready to connect.](./media/smartsheet/signed-in-ready-to-connect.png)
 
-### Connecting to different regions of Smartsheet
+Once the connection is established, you'll see the folders and workspaces you have access to within the **Navigator** dialog. Select the data you want to import and then select **Load**. 
 
-The Smartsheet connector only supports connecting to `https://app.smartsheet.com`. There are currently no plans to support other domains or regions.
+   ![Select data using Navigator.](./media/smartsheet/navigator-view.png)
```
Comment (ptyx507x) [inline powerquery-docs/connectors/smartsheet.md:68]: It feels as if the documentation ends abruptly. You can add a section to contact you or a way to contact support from smartsheet or any sort of FAQ

**Review by ptyx507x — CHANGES_REQUESTED** [review]

File: powerquery-docs/connectors/smartsheet.md (hunk 2 above)
Comment (DougKlopfenstein) [reply powerquery-docs/connectors/smartsheet.md:20]: Updated to the correct Product data.

**Comment by work-insights-data** [comment]
> Hi @work-insights-data - I've edited the article for spelling, grammar, formatting, and Microsoft style. Please take a look at the results and let me know if everything is still correct. Once you approve, I can go ahead and merge the content.

Approved. Please merge.

**Pull request closed without merging**